packages/rum-core/src/common/observers/page-clicks.js

```diff
@@ -96,3 +83,34 @@ function getTransactionMetadata(target) {
 
   return metadata
 }
+
+// builds the name of a transaction from the given target
+// Strategy: use custom html attribute 'data-transaction-name' - otherwise fall back to "tagname" + "name"-Attribute
+function buildTransactionName(target) {
+  // Verify if a custom transaction name has been defined
+  const dtName = findCustomTransactionName(target)
+  if (dtName) {
+    return dtName
+  }
+
+  const tagName = target.tagName.toLowerCase()
+  // "tagname" + "name"-Attribute
+  const name = target.getAttribute('name')
+  if (!!name) {
+    return `${tagName}["${name}"]`
+  }
+
+  // Just use the tagName
+  return tagName.toLowerCase()
```

Comment:
toLowerCase not required. Already lowercased.

Reply:
leftover 🍕